micrometer-core/src/main/java/io/micrometer/core/instrument/binder/build/BuildInfoMetrics.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micrometer.core.instrument.binder.build;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+
+import java.time.Instant;
+
+import static java.util.Objects.requireNonNull;
+
+public class BuildInfoMetrics implements MeterBinder {
+
+    private final BuildInfo buildInfo;
+
+    public BuildInfoMetrics(BuildInfo buildInfo) {
```

Comment:
May be one more constructor to add user provided tags to metrics?

Reply:
This has been [added](https://github.com/micrometer-metrics/micrometer/pull/4286/commits/c467a76710dc0cba22c59596b244737264ab9142).